packages/babel-helpers/src/helpers.js

```diff
@@ -633,28 +603,93 @@ helpers.possibleConstructorReturn = () => template.program.ast`
   }
 `;
 
+helpers.superPropBase = () => template.program.ast`
+  import getPrototypeOf from "getPrototypeOf";
+
+  export default function _superPropReceiver(object, property) {
```

Comment:
Looks like this needs a rename to match the helper name.

Reply:
Done.